Doc/library/asyncio-eventloop.rst

```diff
@@ -191,8 +191,8 @@ Scheduling callbacks
 
 .. method:: loop.call_soon(callback, *args, context=None)
 
-   Schedule a *callback* to be called with *args* arguments at
-   the next iteration of the event loop.
+   Schedule the *callback* :term:`callback` to be called with
```

Comment:
The [documentation](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task) for `asyncio.create_task()` says "Wrap the *coro* coroutine" instead of "Wrap a *coro* coroutine."